content/general/limits.textile

```diff
@@ -401,9 +401,9 @@ Integration limits relate to the rate of "webhooks":/general/webhooks and the ra
 |_=. Limit |_=. Free |_=. Self-service |_=. Business (legacy) |_=. Enterprise |
 | "Webhook batch size":#webhook-batch | 50 | 100 | 100 | Custom |
 | "Webhook batch concurrency":#webhook-concurrency | 1 | 1 | 1 | Custom |
-| "Firehose message rate":#firehose-messages (per second) | - | - | - | Custom |
+| "External queue (Firehose) message rate":#firehose-messages (per second) | 50 | - | - | Custom |
```

Comment:
Small suggestion that we update the related heading name and paragraph for this limit if we're changing the name, but good to go without.

Reply:
Good idea. I'm going to slightly reword to I think.